ui/src/message_popup/scaffolding_card.jsx

```diff
@@ -119,7 +119,7 @@ export default React.createClass({
         {showSlider &&
           <div>
             <div style={styles.optionTitle}>Session Length: {sessionLength} {sessionLength===1 ? "question" : "questions"}</div>
-            <Slider key={selectedIndicatorId} value={sessionLength} min={1} max={questions.length} step={1} onChange={this.onSliderChange}/>
```

Comment:
The reasoning behind having the key set as the `selectedIndicatorId` was so that the slider would be forced to update when the indicator got changed. For some reason, updating the max wont be seen until the next setState. By setting a key that changes, it essentially created a new slider for each indicator which got around the issue of the max not updating. It's possible that the slider not updating is a problem caused elsewhere though.